lsp/java/src/main/java/com/itsaky/androidide/lsp/java/actions/FieldBasedAction.kt

```
@@ -114,9 +116,7 @@ abstract class FieldBasedAction : BaseJavaCodeAction() {
     }
 
     if (type == null) {
-      throw CompletionException(

```

I think it's best to let this throw an exception instead of failing silently, any reason you removed it?